File: 프로그래머스/0/120824. 짝수 홀수 개수/짝수 홀수 개수.py
def solution(num_list):
    answer = []
    even, odd = 0, 0
    
    for num in num_list:
            if(num % 2 == 0):
                even += 1
            else:
                odd += 1
                
    answer.append(even)
    answer.append(odd)
    
    return answer

# 다른 풀이
answer = [0,0]
for n in num_list:
    answer[n%2]+=1
return answer

File: 프로그래머스/0/120824. 짝수 홀수 개수/짝수 홀수 개수.java
class Solution {
    public int[] solution(int[] num_list) {
        int[] answer = new int[2];
        
        int count = 0;
        for(int num : num_list) {
            if(num % 2 == 0) {
                count++;
            }
        }
        
        answer[0] = count;
        answer[1] = num_list.length - count;
        
        return answer;
    }
}